### PR TITLE
Add LeftRightMotors Subsystem

### PR DESCRIPTION
**Pull request by ellazzhang**
This subsystem controls the left and right wheel motors. It can move the robot forwards, move the robot backwards, and make the robot turn.

### CODE REVIEW

File: src/org/usfirst/frc/team1076/robot/subsystems/LeftRightMotors.java [hunk 1]
```diff
@@ -1,14 +1,37 @@
...
-    // here. Call these from Commands.
+	SpeedController leftMotor;
+	SpeedController rightMotor;
+	
+	public LeftRightMotors(SpeedController leftMotor, SpeedController rightMotor) {
+		this.leftMotor = leftMotor;
+		this.rightMotor = rightMotor;		
+	}
+	
+	public void setLeftSpeed(double speed) {
```
Comment (porglezomp) [inline src/org/usfirst/frc/team1076/robot/subsystems/LeftRightMotors.java:30]: It would be good to have doc comments on each of these functions.

**Review by porglezomp — CHANGES_REQUESTED** [review]
Add doc comments

File: src/org/usfirst/frc/team1076/robot/subsystems/LeftRightMotors.java [hunk 2]
```diff
@@ -15,20 +15,31 @@ public LeftRightMotors(SpeedController leftMotor, SpeedController rightMotor) {
 		this.leftMotor = leftMotor;
 		this.rightMotor = rightMotor;		
 	}
-	
+	/**
+	 * sets the speed of the left motor
```
Comment (porglezomp) [inline src/org/usfirst/frc/team1076/robot/subsystems/LeftRightMotors.java]: These look pretty good, but there are two things I would change:
- Document the expected range for `speed` (-1 to 1)
- Capitalize and punctuate all the documentation.

**Review by porglezomp — CHANGES_REQUESTED** [review]
There are some final minor improvements you can make to the documentation, but it's almost there.

**Review by anish2399 — APPROVED** [review]
This code looks good.

**Review by porglezomp — APPROVED** [review]

File: src/org/usfirst/frc/team1076/robot/subsystems/LeftRightMotors.java (hunk 2 above)
Comment (porglezomp) [reply src/org/usfirst/frc/team1076/robot/subsystems/LeftRightMotors.java]: I'm going to have Anish make these changes.

**Review by porglezomp — APPROVED** [review]
These are helpful doc comment improvements, this looks good.

**Pull request merged**